project/dimensions/enduses.csv

```diff
@@ -1,14 +1,15 @@
 id,name,fuel_id,unit
 electricity_computers_and_tvs,Computers and TVs,electricity,MWh
 electricity_dishwasher,Dishwasher,electricity,MWh
+electricity_clothes_washing,Clothes Washing,electricity,MWh
 electricity_drying,Drying,electricity,MWh
 electricity_cooking,Cooking,electricity,MWh
-electricity_cooling_and_heating,Cooling and Heating,electricity,MWh
+electricity_cooling_and_heating,Heating and Cooling,electricity,MWh
 electricity_fans_and_pumps,Fans and Pumps,electricity,MWh
 electricity_lighting,Lighting,electricity,MWh
-electricity_miscellaneous_electrical_loads,Miscellaneous Electrical Loads (MELs),electricity,MWh
-electricity_non-PC_office_equipment,Non-PC Office Equipment,electricity,MWh
-electricity_other,Other,electricity,MWh
+electricity_miscellaneous_electrical_loads,Commercial Miscellaneous Electrical Loads (MELs) & Other,electricity,MWh
```

Comment:
Do you mind if I change the id fields
```
electricity_miscellaneous_electrical_loads -> electricity_commerical_miscellaneous_electrical_loads
electricity_other -> electricity_residential_miscellaneous_electrical_loads
```
?

Reply:
Should we just consolidate these into `electricity_miscellaneous_electrical_loads` since the sector is differentiated by the sector column?

Reply:
I initially had in my head that we'd need to keep these separate (even though there is duplication/redundancy in the sector column) because of some nuances in how the AEO reports commercial "other" electricity use, but I think for simplicity we can lump them together. 

I've updated my buildings data set to use the end-use name `MELs & Other` so if we can change to that then it wouldn't require additional revisions to my data set.